data/human/names.txt

```diff
@@ -3420,7 +3491,7 @@ phrase "trees"
 		"Elder"
 		"Elm"
 		"Eucalyptus"
-		"Ginko"
+		"Ginkgo"
```

Comment:
This seems to be an unrelated change.

Reply:
Yeah, Ginko is a typo. Might as well change it here instead of another PR.